### ISSUE TITLE
Various thoughts and things that need to be improved with dragon fighting

### ISSUE THREAD

**Issue by Kusark** [Alex-the-666/Ice_and_Fire#56]
I was fighting a stage 3 white ice dragon and I noticed multiple things that didn't seem right.

1. The dragon did not use any abilities besides the basic bite attack. I was hoping for some tail attacks or special magic attacks, but the dragon would just constantly rush me and insisted on biting me.

2. The dragon wouldn't fly. That's another thing that I didn't think was right. The dragon would only run towards me and continue to bite me until I finally killed it

3. The dragon's hit box is way too small. I missed so many shots simply because I wasn't aiming at the small box in the center of the dragon. Increasing the hit box size would help a ton.

4. The dragon's AI isn't that smart at the moment. But of course, the mod is in beta so I understand. The dragon targets other mobs during fights (which I know is a current bug) and tends to get itself into odd places - which brings me into my next thought.

5. After I killed the dragon, for some reason it's body sinks into the ground, but the hit box doesn't. By pressing F3+B i was able to see the hit box and loot the remains, yet the dragon's model was around 1.5 blocks underground. That seems to a bug.

      Basically I think the dragons need to fly around more and mix up it's attacks. Overall more flying would make thing's a lot cooler and more immersive. With the higher stage dragons, perhaps have them fly around in the air and have them use their breath attacks to the player on the ground and also have them occasionally land on the ground and use it's melee attacks.

**Comment by raptorfarian** [comment]
Please don't report mechanic suggestions on the Git Hub aswe are currently in a feature freeze.
